utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/encode/AvroSchemaOutputValidator.scala

```diff
@@ -25,7 +26,7 @@ private[encode] case class AvroSchemaExpected(schema: Schema) extends OutputVali
   override def expected: String = AvroSchemaOutputValidatorPrinter.print(schema)
 }
 
-class AvroSchemaOutputValidator(validationMode: ValidationMode) extends LazyLogging {
+class AvroSchemaOutputValidator(schema: Schema, validationMode: ValidationMode) extends LazyLogging with SinkValueValidator {
```

Comment:
Why this schema have to be on class level? It is tricky now, because we pass around nested schemas in private methods.